.github/workflows/tools-buildtest.yml

```diff
@@ -9,6 +9,7 @@ on:
   pull_request:
     branches:
       - master
+      - '[0-9][0-9][0-9][0-9].[0-9][0-9]-branch'
```

Comment:
How about :

```suggestion
      - '*'
```

?

That would work for master and release branch, but also for feature branches that were already used in the past.

Reply:
Good point!

Reply:
Done.